pkgs/games/crawl/default.nix

```diff
@@ -1,29 +1,36 @@
-{ stdenv, lib, fetchFromGitHub, which, sqlite, lua5_1, perl, zlib, pkgconfig, ncurses
+{ stdenv, lib, fetchFromGitHub, which, sqlite, lua5_1, perl, zlib, pkgconfig, ncurses, python, pythonPackages
```

Comment:
is there a better way than adding `pythonPackages` to as a parameter when I only need `pythonPackages.pyyaml` in `nativeBuildInputs`?

Reply:
you can use `python.pkgs.pyyaml` so you only need `python` as a parameter.